src/index.js

```diff
@@ -520,15 +519,23 @@ function createConfig(options, entry, format, writeMeta) {
 							],
 						],
 					}),
-					// Only used for async await
 					babel({
-						// We mainly use bublé to transpile JS and only use babel to
-						// transpile down `async/await`. To prevent conflicts with user
-						// supplied configurations we set this option to false. Note
-						// that we never supported using custom babel configs anyway.
-						babelrc: false,
 						extensions: EXTENSIONS,
 						exclude: 'node_modules/**',
+						presets: [
+							[
+								'@babel/preset-env',
+								{
+									loose: true,
+									modules: false,
+									useBuiltIns: 'usage',
+									corejs: 2,
```

Comment:
I don't think this is needed

Reply:
It is... Sadly. If you upgrade to the latest babel, v3 is implied as the new default, however, it seems that some plugins are still using v2. I'll run some diagnosis in the morning if you want :)

Reply:
Keeping core-js will also only halt rollup due to circular references, please tell me if I’m missing something but as far as I know, it’s meant for polyfilling and consumer building.

Reply:
Oh i thought v2 was the default lets bump it to v3 for sure!

I'm all for disabling core-js. Just need to check if bublé has it and if people can still enable it with there own babelrc file. I'll probably need to create some e2e tests for this ^^